pkg/kubelet/custommetrics/custom_metrics.go

```diff
@@ -38,9 +38,9 @@ func GetCAdvisorCustomMetricsDefinitionPath(container *v1.Container) (*string, e
 	// mounted as a volume. Custom Metrics definition is expected to be in "definition.json".
 	if container.VolumeMounts != nil {
 		for _, volumeMount := range container.VolumeMounts {
-			if path.Clean(volumeMount.MountPath) == path.Clean(CustomMetricsDefinitionDir) {
+			if filepath.Clean(volumeMount.MountPath) == filepath.Clean(CustomMetricsDefinitionDir) {
```

Comment:
does this do the right thing when presented with `/`-delimited mount paths from API data?

Reply:
Reverted until I have more concrete information on this.